src/ray/protobuf/serve.proto

```diff
@@ -139,3 +149,45 @@ message DeploymentVersion {
   string code_version = 1;
   bytes user_config = 2;
 }
+
+message ReplicaConfig {
+  string deployment_def = 1;
+  bytes init_args = 2;
+  bytes init_kwargs = 3;
+  map<string, double> ray_actor_options = 4;
+}
+
+message DeploymentInfo {
+  DeploymentConfig deployment_config = 1;
+  ReplicaConfig replica_config = 2;
+  int32 start_time_ms = 3;
+  string actor_name = 4;
+  bytes serialized_deployment_def = 5;
+  string version = 6;
+  int32 end_time_ms = 7;
+}
+
+message DeploymentRoute {
```

Comment:
Is there a reason we keep this separate from `DeploymentInfo`? Can we put `route` in `DeploymentInfo`? And can we change `DeploymentRouteList` into `DeploymentInfoList`? cc @simon-mo @edoakes

Reply:
In order not to destroy the existing `DeploymentInfo` structure, the `DeploymentInfo` and `route` are encapsulated in the `DeploymentRoute`. Can the integration of deploymentinfo and route be evaluated later?
 @simon-mo

Reply:
I think looks fine for now!